### DISCUSSION TITLE
Command line pear not working after brew install php

### DISCUSSION THREAD

**Discussion by visskiss** [Homebrew/discussions#1549] [Everyday usage]
Hello all,

I have searched everywhere for a solution to my pear extensions seemingly installing via command line and then not appearing when a page loads.

I checked in /usr/local/bin and everything is linked to something in /opt/homebrew/Cellar except for pear, peardev, and pecl.  I tried removing those files and then linking to the appropriate home-brew file but that's not working either.

pear install Log seems to work correctly.

`% pear install Log
WARNING: configuration download directory "/private/tmp/pear/download" is not writeable.  Change download_dir config variable to a writeable dir to avoid this warning
Package "pear.php.net/Log" dependency "pear.php.net/DB" has no releases
Package "pear.php.net/Log" dependency "pear.php.net/MDB2" has no releases
Package "pear.php.net/Log" dependency "pear.php.net/Mail" has no releases
Did not download optional dependencies: pear/DB, pear/MDB2, pear/Mail, use --alldeps to download automatically
pear/Log can optionally use package "pear/DB" (version >= 1.3)
pear/Log can optionally use package "pear/MDB2" (version >= 2.0.0RC1)
pear/Log can optionally use package "pear/Mail"
pear/Log can optionally use PHP extension "sqlite"
downloading Log-1.13.3.tgz ...
Starting to download Log-1.13.3.tgz (46,815 bytes)
.............done: 46,815 bytes
install ok: channel://pear.php.net/Log-1.13.3
`

There is that warning though.

Then:
'% sudo apachectl restart
'

but when I load the page I get "Warning: require_once(Log.php): Failed to open stream..."

Ideas?

**Comment by SMillerDev** [comment]
Pear is deprecated in the php version that homebrew ships.

Aditionally, did you try fixing this?: 
> WARNING: configuration download directory "/private/tmp/pear/download" is not writeable. Change download_dir config variable to a writeable dir to avoid this warning
